### DISCUSSION TITLE
Cannot login -- phone number already used

### DISCUSSION THREAD

**Discussion by xucian** [vercel/community#4013] [Account Issue]
### Question

Hello,

I remember creating an account on vercel, then closing it, but somehow the phone number might've gotten stuck in vercel's database.
I don't know why I deleted the account or even if I deleted it, but I cannot seem to login anymore, either using my email or my github login. 
There's no way of finding out what email is my phone number attached to.

I appreciate any solutions

**Comment by amyegan** [comment]
Hey @xucian. Sorry you're having trouble with registration. Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the details of the project you want to deploy. That will open a new ticket with the team that can help with this. 🙏
